clusterDeployer.py

```diff
@@ -305,7 +306,8 @@ def create_masters(self) -> None:
         # configure DHCP entries for all masters on the local virbr.
         for h in hosts_with_masters:
             h.configure_bridge()
-            h.configure_master_dhcp_entries(self._local_host.bridge)
+        node_configs: list[NodeConfig] = [node.config for h in hosts_with_masters for node in h.k8s_master_nodes]
```

Comment:
nested for loop in 1 statement. We might want to break this apart, as it is difficult to read.

Reply:
Will do in updated PR.